### PR TITLE
feat(update): mention JBoss EAP 8 in the migration guide

### PR DESCRIPTION
**Pull request by yanavasileva**
https://github.com/camunda/camunda-bpm-platform/issues/4142

### CODE REVIEW

File: content/update/minor/721-to-722/_index.md [hunk 1]
```diff
@@ -54,3 +55,47 @@ Before starting, ensure you have downloaded the Camunda 7.22 distribution for th
 
 # Camunda Connect
  We’ve moved the `camunda-connect` project from its [previous location](https://github.com/camunda/camunda-connect) into the [mono repository](https://github.com/camunda/camunda-bpm-platform). We’re no longer versioning it independently. Instead, we’ve integrated it into the 7.X.Y versioning scheme, so you can conveniently declare Camunda `7.22.0-alpha2` to use the latest release of Camunda Connect.
+ 
+# Update to JBoss EAP 8
+
+With this release, we support JBoss EAP 8.0. It Jakarta EE compliant platform.The artifacts are shipped with the latest pre-packaged [Camunda 7 WildFly distribution]({{< ref "/installation/full/jboss/manual.md#setup" >}}).
```
Comment (joaquinfelici) [inline content/update/minor/721-to-722/_index.md:61]:
```suggestion
With this release, we support JBoss EAP 8.0. It Jakarta EE compliant platform. The artifacts are shipped with the latest pre-packaged [Camunda 7 WildFly distribution]({{< ref "/installation/full/jboss/manual.md#setup" >}}).
```

Comment (joaquinfelici) [inline content/update/minor/721-to-722/_index.md:61]: ❓ I'm not sure exactly I understand the second sentence, maybe you meant:

```suggestion
With this release, we support JBoss EAP 8.0, its Jakarta EE compliant platform.The artifacts are shipped with the latest pre-packaged [Camunda 7 WildFly distribution]({{< ref "/installation/full/jboss/manual.md#setup" >}}).
```

**Comment by joaquinfelici** [comment]
It looks great 👍 
I just added one small suggestion and a question in the same paragraph.

**Review by joaquinfelici — COMMENTED** [review]
It looks great 👍
I just added one small suggestion and a question in the same paragraph.

File: content/update/minor/721-to-722/_index.md (hunk 1 above)
Comment (yanavasileva) [reply content/update/minor/721-to-722/_index.md:61]: Yes, thank you.

**Review by joaquinfelici — APPROVED** [review]
Looks good 👍

**Pull request merged**